schema/rss-service-plan-v4.json

```diff
@@ -353,33 +353,12 @@
           "type": "object",
           "required": [
             "address",
-            "dns_servers",
-            "ntp_servers",
             "type"
           ],
           "properties": {
             "address": {
               "type": "string"
             },
-            "dns_servers": {
```

Comment:
(same as all-zones-requests.json) To be clear: the change to this file is okay, even though it's an on-disk interface, because new schema describes a subset of the old schema? i.e., we will be able to continue reading files that had these properties?

Reply:
Yep!